appserver/security/appclient.security/src/main/java/com/sun/enterprise/security/appclient/AppClientSecurityInfoImpl.java

```diff
@@ -56,7 +50,10 @@
 @Service
 public class AppClientSecurityInfoImpl implements AppClientSecurityInfo {
 
-    private static final Logger LOG = System.getLogger(AppClientSecurityInfoImpl.class.getName());
+    private static Logger _logger=null;
```

Comment:
I think here you have to remove your changes.

Reply:
I did, thanks!